v4/source/users.yaml

```diff
@@ -1405,6 +1797,24 @@
             userID := "BbaYBYDV5IDOZFiJGBSzkw1k5u"
 
             mfaSecret, resp = Client.GenerateMfaSecret(userID)
+        - lang: PHP
+          source: |
+            require 'vendor/autoload.php';
+
+            use \Gnello\Mattermost\Driver;
+
+            $container = new \Pimple\Container([
+                "driver" => [
+                    "url" => "https://your-mattermost-url.com",
+                    "login_id" => "email@domain.com",
+                    "password" => "Password1",
+                  ]
+            ]);
+
+            $driver = new Driver($container);
+            $driver->authenticate();
+
+            $result = $driver->getUserModel()->generateMfaSecret("userid");
```

Comment:
0/5 Here I am missing something to identify the value we actually want. This happens on many examples, but this one felt more missing than others. I am not familiar with the `Gnello\Mattermost` code, but I would appreciate adding some line like:
```
if $resut->ok() {
   $mfaSecret = $result->content
}
```
Of course, previous lines adapted to what actually `Gnello\Mattermost` returns.

Reply:
Ok, I'll provide a result manipulation in all the examples. 
It will be a bit verbose due to the nature of the returned object ([here](https://github.com/gnello/php-mattermost-driver#check-results) an example) but I'll do my best.